dist/index.js

```diff
@@ -3690,40 +3691,40 @@ function removeHook (state, name, method) {
 /***/ (function(module, exports) {
 
 "use strict";
-
-
-(function () {
-
-    function chunk (collection, size) {
-    
-        var result = [];
-        
-        // default size to two item
-        size = parseInt(size) || 2;
-        
-        // add each chunk to the result
-        for (var x = 0; x < Math.ceil(collection.length / size); x++) {
-            
-            var start = x * size;
-            var end = start + size;
-            
-            result.push(collection.slice(start, end));
-            
-        }
-        
-        return result;
-        
-    };
-
-    // export in node or browser
-    if (true) {
-        if ( true && module.exports) {
-            exports = module.exports = chunk;
-        }
-        exports.chunk = chunk;
-    } else {}
-
-}.call(this));

```

Comment:
this whole section looks the same?

Reply:
I ever reset the file, and the changes still continue...